src/app/component/project-option/export/export.component.ts

```diff
@@ -32,4 +35,8 @@ export class ExportComponent {
     export() {
         this.printer.print(this.reducedColor, this.usage, this.project, `beadifier_${this.project.boardConfiguration.nbBoardWidth}x${this.project.boardConfiguration.nbBoardHeight}`);
     }
+
+    callback() {
```

Comment:
Same as above, I don't think this is useful.

Reply:
Thanks, updated as per comment above.